config/src/config/secure_config.rs

```diff
@@ -17,6 +17,7 @@ pub struct SecureConfig {
 #[serde(default, deny_unknown_fields)]
 pub struct KeyManagerConfig {
     pub json_rpc_address: SocketAddr,
```

Comment:
why isn't this just a url? that way you don't enter this weirdness.

Reply:
Thanks, David! My thinking was that a url is somewhat "unstructured" when it comes to being written to a config file (i.e., if we just had "json_rpc_url: String" in the config, it's unclear to the reader/viewer what the format of the url is exactly -- e.g., does it need a specific protocol or will a simple ip address work. Does it support http, https, ftp, etc.). Hence why I thought a SocketAddr and a bool would be more structured and leave less room for error. But if you think a simple url string is fine, we can do that too! 😄

Reply:
That's fair, urls suck, but I think we should document instead and use a URL.

Reply:
Sure, done, thanks!